<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setup</class>
 <widget class="QDialog" name="Setup">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>425</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Guido Engine Setup</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="fButtonBox">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>376</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="fDefaultButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>376</y>
     <width>89</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>2</y>
     <width>369</width>
     <height>93</height>
    </rect>
   </property>
   <property name="title">
    <string>Notes spacing</string>
   </property>
   <widget class="QSlider" name="fForceSlider">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>61</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>400</number>
    </property>
    <property name="maximum">
     <number>1500</number>
    </property>
    <property name="pageStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>750</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="fSpringBox">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>27</y>
      <width>63</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>110</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="fForceBox">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>57</y>
      <width>63</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>400</number>
    </property>
    <property name="maximum">
     <number>1500</number>
    </property>
    <property name="value">
     <number>750</number>
    </property>
   </widget>
   <widget class="QLabel" name="fLabel5">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>62</y>
      <width>43</width>
      <height>18</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Force</string>
    </property>
   </widget>
   <widget class="QSlider" name="fSpringSlider">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>31</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="pageStep">
     <number>20</number>
    </property>
    <property name="value">
     <number>110</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="fLabel4">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>32</y>
      <width>47</width>
      <height>18</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Spring</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>212</y>
     <width>371</width>
     <height>77</height>
    </rect>
   </property>
   <property name="title">
    <string>Page layout</string>
   </property>
   <widget class="QCheckBox" name="fNSpacingcheckBox">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>50</y>
      <width>173</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Neighborhood spacing</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fOPFcheckBox">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>26</y>
      <width>145</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Optimum page fill</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>98</y>
     <width>367</width>
     <height>113</height>
    </rect>
   </property>
   <property name="title">
    <string>Systems layout</string>
   </property>
   <widget class="QComboBox" name="fSysDistrMenu">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>78</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>auto</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>always</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>never</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="fMaxDistSlider">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>54</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="fMaxDistBox">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>52</y>
      <width>63</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="fSysDistBox">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>22</y>
      <width>63</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>75</number>
    </property>
   </widget>
   <widget class="QLabel" name="fLabel1">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>27</y>
      <width>123</width>
      <height>18</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>System distance</string>
    </property>
   </widget>
   <widget class="QSlider" name="fSysDistSlider">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>26</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="pageStep">
     <number>50</number>
    </property>
    <property name="value">
     <number>75</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="fLabel2">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>55</y>
      <width>123</width>
      <height>18</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Maximum distance</string>
    </property>
   </widget>
   <widget class="QLabel" name="fLabel3">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>81</y>
      <width>123</width>
      <height>18</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>System distribution</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>290</y>
     <width>371</width>
     <height>79</height>
    </rect>
   </property>
   <property name="title">
    <string>Bounding boxes drawing</string>
   </property>
   <widget class="QCheckBox" name="fPageBB">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>38</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>page</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fSystemBB">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>38</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>systems</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fSystemSliceBB">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>38</y>
      <width>115</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>systems slices</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fStaffBB">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>56</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>staves</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fMeasureBB">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>56</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>measures</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fEventBB">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>56</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>events</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fMapping">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>22</y>
      <width>137</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>include mapping</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fNewMapping">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>22</y>
      <width>137</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>new mapping</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fForceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fForceBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>Setup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>Setup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fForceBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fForceSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fSpringSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fSpringBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fSpringBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fSpringSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fMaxDistSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fMaxDistBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fMaxDistBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fMaxDistSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fSysDistSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fSysDistBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fSysDistBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fSysDistSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
